x-pack/test/reporting/functional/lib/compare_pdfs.js

```diff
@@ -16,6 +16,28 @@ import PDFJS from 'pdfjs-dist';
 
 const mkdirAsync = promisify(mkdirp);
 
+async function convertPageToImage(pdfImage, pageNum, log) {
+  // For some reason the library seems to randomly throw "Failed to convert page to image" errors that are
```

Comment:
You should be able to get rid of all this stuff.  But I think you might need to keep the `denisity` option off, that seemed to be killing `convertPageToImage` on a certain jenkins agent. Namely, `kibana-ci-immutable-ubuntu-1539784860981605281`.  Still have yet to see if it passes on that agent with density turned off.